osgtest/library/condor.py

```diff
@@ -34,3 +34,13 @@ def is_running():
         returncode, _, _ = core.system(['service', 'condor', 'status'])
         return returncode == 0
 
+def wait_for_daemon(collector_log_path, stat, daemon, timeout):
+    """Wait until the requested 'daemon' is available and accepting commands by
+    monitoring the specified CollectorLog from the position specified by 'stat'
+    for a maximum of 'timeout' seconds (default: 60). Returns True if the daemon
```

Comment:
Minor comment: The wait_for_daemon function itself does not provide a default timeout, but rather defers to the monitor_file default.